include/matx/core/type_utils.h

```diff
@@ -473,12 +473,27 @@ template <> struct is_fp16_type<matxFp16> : std::true_type {};
 }
 
 /**
- * @brief Determine if a type is an FF16 type
+ * @brief Determine if a type is an FP16 type
  * 
  * @tparam T Type to test
  */
 template <class T> inline constexpr bool is_fp16_type_v = detail::is_fp16_type<T>::value;
 
+/**
+ * @brief Determine if a type is an FP32 type
+ * 
+ * @tparam T Type to test
+ */
+template<typename T>
+inline constexpr bool is_fp32_type_v = std::is_same_v<typename inner_op_type_t<T>::type, float>;
```

Comment:
you can use `std::is_same_v<float, T>` instead of making a new type trait

Reply:
I had added this for checking if the type was a complex float, since the following was used a couple of times in fft_fftw.h: `std::is_same_v<typename inner_op_type_t<typename OutputTensor::scalar_type>::type, float>`

Reply:
I would change the name to `is_fp32_inner_type` then.

Reply:
Updated